orchagent/aclorch.cpp

```diff
@@ -345,11 +346,28 @@ bool AclRule::create()
     {
         SWSS_LOG_ERROR("Failed to create ACL rule");
         AclRange::remove(range_objects, range_object_list.count);
+        decreaseNextHopRefCount();
```

Comment:
actually you could add a function called increaseNextHopRefCount() similar to the decreaseNextHopRefCount() one so that the ref count could be increased after the SAI API call.